MAUI/Segmented-Control/customization.md

```diff
@@ -0,0 +1,395 @@
+---
+layout: post
+title: Customization of .NET MAUI Segmented control | Syncfusion
+description: Learn about the customization of Syncfusion .NET MAUI Segmented (SfSegmented) control in mobile and desktop applications from a single shared codebase.
```

Comment:
Learn about the appearance customization  of Syncfusion .NET MAUI Segmented Control (SfSegmentedControl)

Reply:
description changed